@Edf2Mat/Edf2Mat.m

```diff
@@ -274,9 +274,9 @@
               ['File ' filename ' not found!']);
             
             obj.filename = filename;
+            [nowine, ~] = system('wine --version');
```

Comment:
please put this line closer to line 293 to comply with High Cohesion => See uncle bobs clean code

Reply:
ok